doc/modules/linear_model.rst

```diff
@@ -923,7 +923,7 @@ for the regularization term :math:`r(W)` via the `penalty` argument:
 +================+==================================================================================+
 | `None`         | :math:`0`                                                                        |
 +----------------+----------------------------------------------------------------------------------+
-| :math:`\ell_1` | :math:`\|W\|_1`                                                                  |
+| :math:`\ell_1` | :math:`\|W\|_{1,1} = \sum_{i,j}|W_{i,j}|`                                        |
```

Comment:
for consistency with the line below put the values of the bounds for the sum. 1 to n and 0 to K-1 although I have no clue why it starts at 1 for i and 0 for j. I would start at 1 for both to stick to maths notations.